ProcessMaker/Events/ScriptDeleted.php

```diff
@@ -13,6 +14,7 @@ class ScriptDeleted implements SecurityLogEventInterface
     use FormatSecurityLogChanges;
 
     private Script $script;
+    public string $userDelete;
```

Comment:
Please remove string $userDelete is not used

Reply:
Solved